src/libraries/System.IO.Compression/src/System/IO/Compression/ZipBlocks.cs

```diff
@@ -534,6 +568,24 @@ public static bool TryValidateBlock(BinaryReader reader, ZipArchiveEntry entry)
 
             return true;
         }
+
+        private static ulong ConvertToUlong(uint lowerAddressValue, uint higherAddressValue)
```

Comment:
Neither this method nor the nothing in this file works with addresses. Why are the arguments called lowerAddress / higherAddress ?

Reply:
Will update to lowerBytes, higherBytes respectively